Bottom: 1f0c479d29f5ee706d6ef97d4d2eda70e849ac0f
Top:    3c7f42f38ff546e0d5456cd17a588ba35985b00f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:23:31 +0000

polygon1.c: Remove now unused memory allocation macro MemGet


---

diff --git a/src/polygon1.c b/src/polygon1.c
index fff6a7a..a5d1325 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -104,10 +104,6 @@ int vect_inters2 (Vector A, Vector B, Vector C, Vector D, Vector S1,
 
 #define error(code)  longjmp(*(e), code)
 
-#define MemGet(ptr, type) \
-  if (UNLIKELY (((ptr) = (type *)malloc(sizeof(type))) == NULL))	\
-    error(err_no_memory);
-
 #undef DEBUG_LABEL
 #undef DEBUG_ALL_LABELS
 #undef DEBUG_JUMP
